### DISCUSSION TITLE
SkipStone disabled ... it has changed...  What is this all about?

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#269] [Q&A] [answered]
Is it just me...  seems like every time I put this Skip project down and pick it up a few days later... I've got another 2 days of figuring out some something... dependencies... licenses... plug-ins...  

Got this Diagnostics about a failed Build:

 Validate plug-in “skipstone” in package “skip”:
    error: “skipstone” was disabled because it has changed (previous fingerprint was 9a537d0890db996075163e36466640cd25625d2c)

Any idea how to make this find a HAPPY PATH?

**Comment by davidakoontz** [comment]
OK... so I started CLEANing every thing...  deleted Derived Data folder... project build folder...  
went looking for the hidden .build folder in Android folder (couldn't find it)...

Finally tried to rebuild in Xcode... another error:  Validate plug-in "slipstone" in package "skip".

**Clicked on some something that wanted to enable skiipstone !!!  <==  This worked.**

Now building this example project to test out new floating menus in Skip.

**Comment by marcprux** [comment, marked as answer]
Every new release of the Skip plugin will trigger Xcode to require approval by the user in order to enable the new plugin version. You should be prompted by Xcode when using it interactively. Are you not seeing this? You can see an example of the dialog at https://forums.swift.org/t/some-build-plug-ins-are-disabled-because-they-have-changed-or-have-never-been-enabled-enable-them-now/61075

If you are building from the command-line with xcodebuild, you can pass the command-line flag `-skipPackagePluginValidation` to bypass the check. You can read more about this at https://forums.swift.org/t/telling-xcode-14-beta-4-to-trust-build-tool-plugins-programatically/59305/11
